plugins/amdgpu/amdgpu_plugin.c

```diff
@@ -521,7 +521,7 @@ int amdgpu_plugin_init(int stage)
 		getenv_bool("KFD_NUMA_CHECK", &kfd_numa_check);
 		getenv_bool("KFD_CAPABILITY_CHECK", &kfd_capability_check);
 	}
-	kfd_max_buffer_size = 0;
+	kfd_max_buffer_size = 4 << 30;
```

Comment:
amdgpu_plugin.c: In function 'amdgpu_plugin_init':
50499
amdgpu_plugin.c:524:33: error: result of '4 << 30' requires 34 bits to represent, but 'int' only has 32 bits [-Werror=shift-overflow=]
50500
  524 |         kfd_max_buffer_size = 4 << 30;

Reply:
@fdavid-amd I think size_t can be 32-bits depending on the compiler used. So probably safer to change 
size_t kfd_max_buffer_size;
to:
uint64_t kfd_max_buffer_size;
and most of the occurrences where we use size_t to uint64_t

Reply:
Mark the symbol as static If the symbol is used only in amdgpu_plugin.c

Reply:
@fdavid-amd any ETA on these changes?